### ISSUE TITLE
md-tabs: animations missing for tab panels

### ISSUE THREAD

**Issue by shhdharmen** [material-components/material-web#5373]
### What is affected?

Component

### Description

When I switch tabs, I would expect tab-panels to animate the same way (but opposite direction) as `md-primary-tab` or `md-secondary-tab`, however it does not happen.

### Reproduction

Check the official playground at https://material-web.dev/components/tabs/stories/, no tab-panels are animated, they just appear/dis-appear.

### Workaround

I have not found a workaround

### Is this a regression?

No or unsure. This never worked, or I haven't tried before.

### Affected versions

1.1.1

### Browser/OS/Node environment

Browser: Microsoft Edge Version 120.0.2210.91 (Official build) (64-bit)

**Comment by asyncLiz** [comment]
There are [examples of this in spec](https://m3.material.io/components/tabs/guidelines#a10c71e1-a038-4ff7-8e2f-1ece2a34e15c), but I think this may be part of the "add tab panels" feature we want to support.

For those tab panels, we could add built-in animation support as panels change.

**Comment by shhdharmen** [comment]
@asyncLiz correct, I was asking for built-in animation. Because other components in material-web, like dialogues, have the built-in animation.
